inc/enqueue.php

```diff
@@ -51,6 +51,26 @@ function bootscore_scripts() {
 add_action('wp_enqueue_scripts', 'bootscore_scripts');
 
 
+
+/*
+ * Register compiled CSS to editor
+ */ 
+function bootscore_add_editor_styles() {
+  add_theme_support('editor-styles');
+  add_editor_style('assets/css/main.css');
+
+  // Check if the current page is the Gutenberg editor and enqueue CSS to tha main admin area to use variables in theme.json
+  $screen = get_current_screen();
+  if ($screen && $screen->is_block_editor) {
+    wp_enqueue_style('main-editor-style', get_stylesheet_directory_uri() . '/assets/css/main.css', array(), '1.0', 'all');
+    // Fix minor CSS issues in the backend editor screen
+    wp_enqueue_style('admin-style', get_template_directory_uri() . '/assets/admin/css/admin.css', array(), '1.0', 'all');
```

Comment:
> We can make a separate css file which doesn't have the reboot styling so we only focus on the components.

Yes, any idea?

> Shouldn't we use filemtime here for the version?

Yes, we can

Reply:
> Yes, any idea?

If we make a new scss file, we exclude some of the parts of Bootstrap easily. Just need to be aware that this can raise the loading times because more scss is compiled, so we need to add a way that it'll skip some compiler checks... May need to expand the SCSS compiler before we can achieve that.